README.md

```diff
@@ -1,2 +1,80 @@
 # ecoCode-rule-validator-js-cadvisor
+
 JS Rules validator using cadvisor
+
+## Install prerequisites
+
+You can run the code in a [VSCODE devcontainer](https://code.visualstudio.com/docs/devcontainers/containers), which will launch the [PostGres](https://www.postgresql.org/) database and the metrics service [cAdvisor](https://github.com/google/cadvisor)
+
+You need to have [Docker](https://www.docker.com/), [Node](https://nodejs.org/en/), [VSCode](https://code.visualstudio.com/), and the [Dev Container extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) installed on your machine.
```

Comment:
The version for the tools are either specified in the docker-compose, or the package.json for Node itself.

I'm using the latest versions for cAdvisor and Postgres because it does not really matter, but it could be changed for fixed versions if you'd like.

Concerning docker, the compose format is specified in the file itself.